q3tiledaligner/src/au/edu/qimr/tiledaligner/GenerateTiledAlignerFile.java

```diff
@@ -67,33 +75,29 @@ void performBinning() throws IOException {
 				break;
 			}
 			
-			chrLengthPosition.add(currentChr + ":" + referenceBasesLength + ":" + longPosition); 
+			chrLengthPosition.add(currentChr + Constants.COLON + referenceBasesLength + Constants.COLON + longPosition); 
```

Comment:
For my education only, what does `Constants.COLON` etc. give you that a literal ":" does not?

Reply:
Constants.COLON = ":"

Reply:
I think this harks back to the days before Java would intern strings. Perhaps using a static variable removed the need for the lookup in the String pool...